### DISCUSSION TITLE
Any way to start from scratch?

### DISCUSSION THREAD

**Discussion by P1xelC0da** [cockpit-project/cockpit#20339] [Q&A] [answered]
I just installed Cockpit and jacked up the home page / login. Is there some way to delete cockpit to wipe all the customizations?

**Comment by martinpitt** [comment]
It depends on what you did -- if you edited cockpit's files, then uninstall or reinstall all packages (debs/rpms). Cockpit stores a few settings (login cookie, language, known remote hosts) in the browser's localstore, so you can delete settings for "localhost" in your browser. Or try a private window first if that actually fixes it.

**Comment by P1xelC0da** [comment]
Good evening.

I was able to get it fixed, however when I go to authenticate as sudo and type in my password (before entering) it returns "Problem becoming administrator" and makes me log back in entirely.

**Reply by martinpitt** [reply]
Diagnosing this requires at least a `journalctl` output.

**Comment by P1xelC0da** [comment, marked as answer]
It’s the CSS settings I screwed up, lol.
